package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "react-redux-todo-app",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/Kartalmiseban01/react-redux-todo-app.git",
+  "author": "Eban Ömer KARTALMIŞ <kartalmiseban@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "start": "parcel src/index.html"
```

You must define commands for parcel.js bundler to your react project like: `start`, `dev`, `build`
```suggestion
    "start": "parcel src/index.html",
        "start": "cross-env NODE_ENV=development parcel src/index.html --no-cache",
    "build":
      "cross-env NODE_ENV=production parcel build src/index.html --public-url ./"
```
You want to specify the environment, you must install the package:
`yarn add  -D cross-env`